v19.2/explain.md

```diff
@@ -72,15 +77,17 @@ By default, `EXPLAIN` includes the least detail about the query plan but can be
 ~~~
 
 ~~~
-    tree    | field  |   description
-+-----------+--------+------------------+
-  sort      |        |
-   │        | order  | +season
-   └── scan |        |
-            | table  | episodes@primary
-            | spans  | ALL
-            | filter | season > 3
-(6 rows)
+    tree    |    field    |   description
++-----------+-------------+------------------+
+            | distributed | true
+            | vectorized  | false
```

Comment:
Isn't it the case that vectorized execution is also farmed out/distributed across nodes? At least that's what I read on the 'Vectorized Query Execution' page (AIUI). Unfortunate terminology situation arising here.  FYI @jordanlewis @awoods187 for future thinking about these terms (unless I'm totes misunderstanding this?)
Also, AFAICT we have not documented these fields anywhere, so we should do that.  Also the definition of "Field" needs to be updated since it says "name of a parameter relevant to the query plan node immediately above", but there is no plan node above these.